Documentation/CRDs/Block-Storage/ceph-block-pool-crd.md

```diff
@@ -28,41 +34,30 @@ spec:
   deviceClass: hdd
 ```
 
-#### Hybrid Storage Pools
+### Erasure Coded RBD Pool
 
-Hybrid storage is a combination of two different storage tiers. For example, SSD and HDD.
-This helps to improve the read performance of cluster by placing, say, 1st copy of data on the higher performance tier (SSD or NVME) and remaining replicated copies on lower cost tier (HDDs).
+This example specifies a pool with an EC 4+2 data protection scheme.  For a
+given amount of raw storage, this pool yields double the usable space compared
+to the replicated ``size 3`` example above.  Redundancy is provided by
+specifying an [erasure coding](#erasure-coding) profile with ``codingChunks: 2``,
+akin to a conventional RAID6 volume but with much greater flexibility.  Data
+remains available if any one node is down and is preserved if any two are lost.
```

Comment:
Since the example sets `failureDomain: osd`. (edit: ignore this suggestion if we change the example to host failure domain)
```suggestion
remains available if any one OSD is down and is preserved if any two are lost.
```

Reply:
Ack.  Honestly the concept of OSD failure domain is so alien to me that I struggled a bit with this section.  Pushing a change.